packages/rx_bloc_cli/mason_templates/bricks/feature_onboarding/API_CONTRACTS.md

```diff
@@ -0,0 +1,274 @@
+# Onboarding/Registration API Contracts
```

Comment:
Please add an onboarding section in the main README.md file, which is visible here - https://pub.dev/packages/rx_bloc_cli/example with a link the the specification

Reply:
Added onboarding section. Let me know if any additional changes are needed.